connector_ebisumart/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+This module connects Odoo with Ebisumart.
+Ebisumart (https://ebisumart.com/) is a popular Japanese e-commerce platform.
+
+This module allows the synchronization of the following objects from Ebisumart to Odoo:
+- Product
+- Partner
+- Sale Order
+- Purchase Order
```

Comment:
It's misleading to include purchase orders (they don't come from Ebisumart; they are generated within Odoo).

@kanda999 Please confirm about partners as my memory is vague.